src/WebJobs.Script/Rpc/LanguageWorkerChannel.cs

```diff
@@ -277,13 +284,47 @@ internal void PublishRpcChannelReadyEvent(RpcEvent initEvent)
                 return;
             }
 
-            RpcChannelReadyEvent readyEvent = new RpcChannelReadyEvent(_workerId, _workerConfig.Language, this, _initMessage.WorkerVersion, _initMessage.Capabilities);
-            _eventManager.Publish(readyEvent);
+            var managedDepedencyProvider = new ManagedDependencyProvider();
+            var managedDependencies = managedDepedencyProvider.GetDependencies(Path.Combine(_rootScriptPath, ScriptConstants.HostMetadataFileName), SystemEnvironment.Instance, _managedDependencyOptions);
+            string managedDependencyCapability;
+            if (_initMessage.Capabilities.TryGetValue(EnumDependencies.DependenciesInstalltionEnabled.ToString(), out managedDependencyCapability)
+                && string.Equals(managedDependencyCapability, EnumDependencies.True.ToString(), StringComparison.InvariantCultureIgnoreCase)
```

Comment:
As we now have a Well Known features list, please add an Enum to the proto file. 
```
enum Features {
    ManagedDependencies = 0;
  }
```
Update WorkerInitRequest and Response to include list of features.
This code will then become simpler, check if the list of features contains enum ManagedDenpendencies